### ISSUE TITLE
Error starting

### ISSUE THREAD

**Issue by drakmail** [Turistforeningen/sentry#1]
When I run

```
$ docker-compose up
```

I got this error:

```
Couldn't find env file: /home/sentry/sentry/private-data/environment.sh
```

**Comment by Starefossen** [comment]
Yes, this is unfortunate but this setup requires an environment file with your personal/secret configurations. 

I'll see if I can post which environment variables are required to get Sentry up and running.

**Comment by drakmail** [comment]
It will be awesome if you can provide sample `environment.sh` file

**Comment by jamesob** [comment]
Or at least update the README :)
